script/build-win32.sh

```diff
@@ -10,13 +10,24 @@ if [[ -z "${DESTINATION}" ]]; then
   exit 1
 fi
 
-if [ "$TARGET_ARCH" = "x64" ]; then
-  DEPENDENCY_ARCH="amd64"
-  MINGW_DIR="mingw64"
-else
-  DEPENDENCY_ARCH="x86"
-  MINGW_DIR="mingw32"
-fi
+case $TARGET_ARCH in
+  x64)
+    DEPENDENCY_ARCH="amd64"
+    MINGW_DIR="mingw64"
+    ;;
+  x86)
+    DEPENDENCY_ARCH="x86"
+    MINGW_DIR="mingw32"
+    ;;
+  arm64)
+    DEPENDENCY_ARCH="arm64"
+    MINGW_DIR="clangarm64"
+    ;;
+  *)
+	  echo "Unsupported architecture: $TARGET_ARCH"
+    exit 1
+	  ;;
```

Comment:
Indentation nit (nitdentation? 🤔 🤣 👀):
```suggestion
    echo "Unsupported architecture: $TARGET_ARCH"
    exit 1
    ;;
```